notebooks/114-simplified-mode/114-simplified-mode.ipynb

```diff
@@ -0,0 +1,179 @@
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "model = torch.hub.load(\"chenyaofo/pytorch-cifar-models\", \"cifar10_resnet20\", pretrained=True)\n",
+    "model.eval()\n",
+    "dummy_input = torch.randn(1, 3, 32, 32)\n",
+    "\n",
+    "onnx_model_path = Path(MODEL_DIR) / 'cifar10_resnet20.onnx'\n",
+    "ir_model_xml = onnx_model_path.with_suffix('.xml')\n",
+    "ir_model_bin = onnx_model_path.with_suffix('.bin')\n",
+    "\n",
+    "torch.onnx.export(model, dummy_input, onnx_model_path)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now we convert this model into the OpenVINO IR using the Model Optimizer"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!mo --framework=onnx --data_type=FP32 --input_shape=[1,3,32,32] -m $onnx_model_path  --output_dir $MODEL_DIR"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "In this step we need to write the optimization config and export it to .json file"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "config = dict({\n",
+    "    \"model\": {\n",
+    "        \"model_name\": \"resnet20\",\n",
+    "        \"model\": str(ir_model_xml),\n",
+    "        \"weights\": str(ir_model_bin)\n",
+    "    },\n",
+    "\n",
+    "    \"engine\": {\n",
+    "        \"type\": \"simplified\",\n",
+    "        \"data_source\": DATA_DIR\n",
+    "    },\n",
+    "\n",
+    "    \"compression\": {\n",
+    "        \"target_device\": \"ANY\",\n",
+    "        \"algorithms\": [\n",
+    "            {\n",
+    "                \"name\": \"DefaultQuantization\",\n",
+    "                \"params\": {\n",
+    "                    \"preset\": \"performance\",\n",
+    "                    \"stat_subset_size\": 300\n",
+    "                }\n",
+    "            }\n",
+    "        ]\n",
+    "    }\n",
+    "})\n",
+    "\n",
+    "with open(\"config.json\", \"w\") as outfile:\n",
+    "    json.dump(config, outfile)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now we can optimize the model using this configuration file by calling the following command"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!pot -c config.json"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "In the next releases the following method for using the simplified mode quantization without writing a POT config will be available"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!pot -q default -m $ir_model_xml -w $ir_model_bin --engine=simplified --data-source=$DATA_DIR"
+   ]
+  }
+ ],
```

Comment:
I believe we should add performance measurements at least. I'm not sure about the accuracy as on the one hand we want to avoid AC and on the other hand it would be nice to show the accuracy.